opm/simulators/flow/FlowProblem.hpp

```diff
@@ -343,8 +343,39 @@ class FlowProblem : public GetPropType<TypeTag, Properties::BaseProblem>
     void finishInit()
     {
         ParentType::finishInit();
-
         auto& simulator = this->simulator();
+
+        bool transmissiblity_updated = false;
+        auto processTransmissibilities = [&simulator, &transmissiblity_updated, this]() {
+            // Re-ordering in case of ALUGrid
+            auto gridToEquilGrid = [&simulator](unsigned int i) -> unsigned int {
+                return simulator.vanguard().gridIdxToEquilGridIdx(i);
+            };
+
+            transmissibilities_.finishInit(gridToEquilGrid);
+            transmissiblity_updated = true;
+        };
```

Comment:
I think I'd prefer to write this **slightly** differently in order to avoid the mutable `transmissibility_updated` object with a very large scope:
```C++
auto finishTransmissibilities = [updated = false, this]() mutable
{
    if (updated) { return; }
    this->transmissibilities_.finishInit([&vg = this->simulator().vanguard()](const unsigned int it)
        return vg.gridIdxToEquilGridIdx(i);
    }
    updated = true;
};
```
That way you can unconditionally call `finishTransmissibilities()` and the lambda will do the right thing.

Reply:
This is a very good suggestion.  Incorporated already.